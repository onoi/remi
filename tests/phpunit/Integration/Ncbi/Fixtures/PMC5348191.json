{
    "header": {
        "type": "esummary",
        "version": "0.3"
    },
    "result": {
        "uids": [
            "5348191"
        ],
        "5348191": {
            "uid": "5348191",
            "pubdate": "2017 Mar 10",
            "epubdate": "2017 Mar 10",
            "printpubdate": "",
            "source": "Medicine (Baltimore)",
            "authors": [
                {
                    "name": "Markun S",
                    "authtype": "Author"
                },
                {
                    "name": "Scherz N",
                    "authtype": "Author"
                },
                {
                    "name": "Rosemann T",
                    "authtype": "Author"
                },
                {
                    "name": "Tandjung R",
                    "authtype": "Author"
                },
                {
                    "name": "Braun RP",
                    "authtype": "Author"
                }
            ],
            "title": "Mobile teledermatology for skin cancer screening: A diagnostic accuracy study",
            "volume": "96",
            "issue": "10",
            "pages": "e6278",
            "articleids": [
                {
                    "idtype": "pmid",
                    "value": "28272243"
                },
                {
                    "idtype": "doi",
                    "value": "10.1097/MD.0000000000006278"
                },
                {
                    "idtype": "pmcid",
                    "value": "PMC5348191"
                }
            ],
            "fulljournalname": "Medicine",
            "sortdate": "2017/03/10 00:00",
            "pmclivedate": "2017/03/22"
        }
    }
}