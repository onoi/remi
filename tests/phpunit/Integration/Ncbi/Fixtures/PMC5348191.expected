{
   "ncbi-dbtype":"pmc",
   "pmid":"28272243",
   "doi":"10.1097\/MD.0000000000006278",
   "pmcid":"PMC5348191",
   "author":[
      "Markun S",
      "Scherz N",
      "Rosemann T",
      "Tandjung R",
      "Braun RP"
   ],
   "title":"Mobile teledermatology for skin cancer screening: A diagnostic accuracy study",
   "journal":"Medicine",
   "pubdate":"2017 Mar 10",
   "volume":"96",
   "issue":"10",
   "pages":"e6278",
   "retrieved-from":"http:\/\/www.ncbi.nlm.nih.gov\/",
   "year":"2017",
   "abstract":"Abstract Skin cancer screening has undoubted potential to reduce cancer-specific morbidity and mortality. Total-body exams remain the prevailing concept of skin cancer screening even if effectiveness and value of this method are controversial. Meanwhile, store and forward teledermatology was shown to be a reliable instrument for several diagnostic purposes mostly in specialized dermatology settings. The objective of this study was to evaluate most convenient mobile teledermatology interventions as instruments for skin cancer screening in a representative population. Prospective diagnostic study with visitors of a skin cancer screening campaign in Switzerland. Histopathology was used as reference standard. Mobile teledermatology with or without dermoscopic images was assessed for performance as a screening test (i.e., rule-in or rule-out the need for further testing). Outcomes were sensitivity, specificity, and predictive values. Seven cases of skin cancer were present among 195 skin lesions. All skin cancers were ruled-in by teledermatology with or without dermoscopic images (sensitivity and negative predictive value 100%). The addition of dermoscopic images to conventional images resulted in higher specificity (85% vs. 77%), allowing reduction of unnecessary further testing in a larger proportion of skin lesions. Store and forward mobile teledermatology could serve as an instrument for population-based skin cancer screening because of favorable test performance.",
   "type":"research-article"
}